brainglobe_atlasapi/atlas_generation/atlas_scripts/axolotl_atlas.py

```diff
@@ -0,0 +1,267 @@
+__version__ = "0"
+
+import csv
+import glob as glob
+import time
+from pathlib import Path
+
+import numpy as np
+import pooch
+import tifffile
+from rich.progress import track
+
+from brainglobe_atlasapi import utils
+
+# from skimage import io
+from brainglobe_atlasapi.atlas_generation.mesh_utils import (
+    Region,
+    create_region_mesh,
+)
+from brainglobe_atlasapi.atlas_generation.wrapup import wrapup_atlas_from_data
+from brainglobe_atlasapi.structure_tree_util import get_structures_tree
+
+
+def create_atlas(working_dir, resolution):
+    ATLAS_NAME = "axolotl"
```

Comment:
```suggestions
ATLAS_NAME = "unam_axolotl"
```
Our atlas name usually starts with the institution that made the atlas, in this case UNAM :)

Reply:
FYI it doesn't have to (there's no specific convention), but in this case I think it's a good name!